backend/src/main/java/com/htecgroup/skynest/service/BucketService.java

```diff
@@ -1,10 +1,15 @@
 package com.htecgroup.skynest.service;
 
+import com.htecgroup.skynest.model.dto.BucketDto;
 import com.htecgroup.skynest.model.response.BucketResponse;
 
 import java.util.UUID;
 
 public interface BucketService {
 
   BucketResponse getBucket(UUID uuid);
+
+  BucketDto findBucketById(UUID uuid);
```

Comment:
What is the correct returned mode? you have BucketResponse and BucketDto ? try to be consistent as in the other services

Reply:
BucketResponse is the correct one who is connected to the controller, but I use BucketDto for the delete implementation and for easier test, the same thing is done in the user service as well.